slatedb/src/admin.rs

```diff
@@ -41,6 +43,22 @@ pub struct Admin {
     pub(crate) rand: Arc<DbRand>,
 }
 
+/// Rounding policy for non-exact sequence/timestamp lookups.
```

Comment:
curious why we can't reuse the `FindOption` enum? We can add the other traits to it if needed.